.github/workflows/visual-diff.yml

```diff
@@ -7,8 +7,25 @@ on:
       - completed
 
 jobs:
+  changes:
```

Comment:
Adding this workflow here so we can check on the condition in the `visual-diff` check. (Specifically -> `needs.changes.outputs.backend_or_frontend == 'true'`.)